src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static final int MAX_VALUE_OF_PARAMETER = 25;
+
+    public static final int DEFAULT_RADIUS = 10;
+
+    public static final String DEFAULT_COLOR = "White";
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private final Random random = new Random();
+
+    public Figure getRamdomFigure() {
+        int index = random.nextInt(5);

```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)